src/grid.py

```diff
@@ -1,5 +1,5 @@
 import threading
-from shape import Shape
+from shape import Shape, create_shape
```

Comment:
Probably we are not using this import we should delete create shape from imports